suite-common/message-system/schema/config.schema.v1.json

```diff
@@ -363,7 +363,7 @@
             "title": "Localization",
             "description": "A multilingual text localization.",
             "type": "object",
-            "required": ["en-GB", "en", "es", "cs", "ru", "ja"],
+            "required": ["en-GB", "en", "es", "cs", "ru", "ja", "hu", "it"],
```

Comment:
maybe we should leave the new languages as optional, so we have a chance to simply remove or not use them in the future? What do you think, @tomasklim?

Reply:
Maybe official languages should be required and community can be optional?

Reply:
that's what I thought too, but I would definitely leave what we already have marked as required, just mark new as required only if it become officially supported